### ISSUE TITLE
[WebToolsE2E][Aspire] The dashboard theme is set to light by default. After manually selecting the system theme, closing and reopening Settings, the theme is automatically set to light again.

### ISSUE THREAD

**Issue by v-reinawang** [dotnet/aspire#6282]
**REGRESSION INFO**: Worked in Aspire 8.2.1

**INSTALL STEPS**

1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 9.0.100-rc.2.24474.11
3. Install Aspire 9.0 [9.0.0-preview.5.24513.1]

   - Add dotnet8 feed
   - dotnet new install Aspire.ProjectTemplates::9.0.0-preview.5.24513.1 --force

**REPRO STEPS**

1. Open CMD, run following command to create an Aspire Starter App project
```
dotnet new aspire-starter-9 -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
   `dotnet run`

**NOTE**: After manually selecting the dark theme, closing and reopening Settings, the theme is still dark.

**ACTUAL**:
The dashboard theme is set to light by default. After manually selecting the system theme, closing and reopening Settings, the theme is automatically set to light again.
![Image](https://github.com/user-attachments/assets/033ce082-cddc-4a22-b8bc-368a258a48b7)

**EXPECTED**:
The dashboard theme is set to system by default. After manually selecting the system theme, close and reopen Settings, the theme is still system.

**Comment by JamesNK** [comment]
Is this different to https://github.com/dotnet/aspire/issues/6020?

This issue has been logged 5(?) times.

**Comment by v-reinawang** [comment]
Similar to https://github.com/dotnet/aspire/issues/6020

**Comment by v-reinawang** [comment]
Verify this issue has been fixed in Aspire 9.0.0-rc.2.24517.3
